src/Packager.php

```diff
@@ -132,6 +173,20 @@ private function setConfig(array $config)
         $this->setOrganization($config['organization'] ?? '');
         $this->setPackageFilename($config['packageFilename'] ?? $config['identifier']);
         $this->setCreateZipArchive($config['createZipArchive'] ?? true);
+        $this->setMetadataDescription($config['metadataDescription'] ?? null);
+    }
+
+    /**
+     * Applying metadataConfig
+     *
+     * @param array $metadataConfig
+     */
+    private function setMetadataConfig(array $metadataConfig)
```

Comment:
You have to use null type hint for the first argument. There will be an type exception if you will have pushed null as first argument for this function.